clients/drcachesim/common/options.cpp

```diff
@@ -374,6 +374,18 @@ droption_t<std::string> op_raw_compress(
     "for an SSD, zlib and gzip typically add overhead and would only be used if space is "
     "at a premium; snappy_nocrc and lz4 are nearly always performance wins.");
 
+droption_t<std::string> op_trace_compress(
+    DROPTION_SCOPE_FRONTEND, "compress", DEFAULT_TRACE_COMPRESSION_TYPE,
+    "Trace compression: \"gzip\",\"zlib\",\"lz4\",\"none\"",
+    "Specifies the compression type to use for trace files: \"zip\", "
+    "\"gzip\", \"lz4\", or \"none\".  When it comes to storage types, the impact "
+    "on overhead varies: "
+    "for SSDs, zip and gzip often increase overhead and should only be chosen "
+    "if space is limited; lz4 compression generally improves performance "
+    "and is recommended in most cases."
```

Comment:
nit: " in most cases where fast skipping is not needed".  (So then probabaly should move last sentence up before the "When it comes".)